utils/proc-macro-server-api/Cargo.toml

```
@@ -0,0 +1,14 @@
+[package]
+name = "proc-macro-server-api"
```

```suggestion
name = "scarb-proc-macro-server-types"
```

just like `lsp_types`, wdyt?